tests/unit/src/test/resources/definition/example/NamedArguments.scala

```diff
@@ -22,8 +22,8 @@ object NamedArguments/*NamedArguments.scala*/ {
 
   // anonymous classes
   @deprecated/*deprecated.scala*/(
-    message/*<no symbol>*/ = "a",
-    since/*<no symbol>*/ = susan/*NamedArguments.scala*/
+    message/*deprecated.scala fallback to scala.deprecated#*/ = "a",
```

Comment:
So this is fixed, but the main issue I tried to fix needed to be reopnened unfortunately https://github.com/scalameta/scalameta/issues/2144